nevergrad/optimization/recaster.py

```diff
@@ -237,7 +237,10 @@ def _internal_provide_recommendation(self) -> base.ArrayLike:
         """Returns the underlying optimizer output if provided (ie if the optimizer did finish)
         else the best pessimistic point.
         """
-        assert self._messaging_thread is not None, 'Optimization was not even started'
+        if self._messaging_thread is None:  #Optimization was not even started
```

Comment:
If the thread is None then that means the current best is just 0 right?
Also, related PR/Issue #444 #448